### ISSUE TITLE
Update the CAN hardware filter configurator to match the new transport layer specification

### ISSUE THREAD

**Issue by pavel-kirienko** [UAVCAN/libuavcan#49]
Putting this on hold as it doesn't block the release.

@ilia-sheremet Would be awesome if you could have a look.

![frames](https://cloud.githubusercontent.com/assets/3298404/8510643/32d605be-22fc-11e5-9afc-530299849427.png)

**Comment by ilia-sheremet** [comment]
Got it!

**Comment by ilia-sheremet** [comment]
Done

**Comment by pavel-kirienko** [comment]
Reopening temporarily, will close once the PR is merged.
